FirstUsage.md

```diff
@@ -24,18 +24,70 @@ To use markdown the documentation says we have to `pip install recommonmark` and
 ```python
 extensions = ['recommonmark']
 ```
-# Testing using tox https://tox.readthedocs.io/en/latest/
+# Building package Locally
+First create a `setup.py` file
+```python
+from setuptools import setup, find_packages
+
+setup(
+    name='mcppy',
+    version='0.0.1',
+    description='Python module for CP',
+    author='Tias Guns',
+    author_email='',
+    license='MIT',
+    # packages=find_packages(exclude=['sample_app.test']),
+    install_requires=['numpy'],
+    # entry_points={
+    #     'console_scripts': [
+    #         'sapp=sample_app.sample_app:main'
+    #     ]
+    # }
+)
+```
+Specify the requirements in the requirements.txt file
+```
+numpy>=1.20.0
+```
+Create a folder named `CPModules` (or whatever) and write all the class and definition inside it within `model.py` (or whatever).
+Now to install in test mode, first create a virtual environment. and then run after going to directory containing the `setup.py`
+```
+pip install -e .
+```
+To check whether the installation is successful, open python
+```python
+form CPModules.model import *

```

Comment:
should be 'from cpmpy import *'?

Reply:
It's the name of the folder which in this case `cpmodules`, I think a good practice is to give it a same name as the package, but I d don't know